ts/RailroadMap.ts

```diff
@@ -1021,7 +1028,7 @@ export class RailroadMap {
             const paths = Object.entries(industrySvgPaths[industryName] ?? {});
             paths.forEach(renderPath(g));
         } else {
-            g.text(String(industry.type))
+            g.text(String(industry.type)) // embix: how to enable/test this?
                 .attr('transform', 'rotate(90)')
```

Comment:
You should be able to do something like this:
```suggestion
                .attr('transform', this.invert ? 'rotate(-90)' : 'rotate(90)')
```